oak-query-spi/src/main/java/org/apache/jackrabbit/oak/spi/query/package-info.java

```diff
@@ -18,7 +18,7 @@
 /**
  * This package contains oak query index related classes.
  */
-@Version("1.8.0")
+@Version("2.0.0")
```

Comment:
is this pacakge supposed to be used outside Oak? Or can we mark it as "@Internal"?

Reply:
I guess this is really an SPI anyone could implement.